.github/actions/dependabot-missing-actions-check/action.yml

```diff
@@ -0,0 +1,12 @@
+name: dependabot-coverage-check
+author: wojciech.piotrowiak
+description: Ensure that all GitHub Actions defined in the current repo have managed dependabot updates. Workaround for https://github.com/dependabot/dependabot-core/issues/6345
+runs:
+  using: composite
+
+  steps:
+    - name: Checkout code
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
I would leave that out because most of the time the code will already be checked out on the expected branch/ref to check, we don't want to have to add a boolean input to disable checkout like we do on some actions doing checkout unnecessarily

```suggestion
```